fact_extractor/plugins/unpacking/broadcom/test/test_plugin_broadcom_sao.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from test.unit.unpacker.test_unpacker import TestUnpackerBase
+
+TEST_DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
```

Comment:
We try to replace os.path with the more modern pathlib where possible

```suggestion
TEST_DATA_DIR = Path(__file__).parent / 'data'
```